test_examples/helpers/vpc/main.tf

```diff
@@ -0,0 +1,20 @@
+module "vpc" {
```

Comment:
Can't we use the networking module for this?

Reply:
We can. If we do that, it would be more like an integration test rather than unit test. I'm trying to bring less dependency as possible of our own modules. But if we think having an integration here is good, we could update this or even add another test for that.